src/validify/inputValidation.js

```diff
@@ -0,0 +1,12 @@
+import printInputLengthError from "../message/error/printInputLengthError.js";
+import nanValidation from "./nanValidation.js";
+
+const inputValidation = (arrayInput) => {
+    nanValidation(arrayInput);
+
+    if(arrayInput.length !== 3) {
+        printInputLengthError();
+    }
+}
```

Comment:
1. 함수명은 명사형 대신 동사형으로 짓는것이 명확한 의도와 목적을 전달하는데 더 좋은 것 같아요 ..! 
`ValidateInput` 같은 네이밍은 어떨까요?!
2. 입력값의 유효성을 검증하는 작업은 (1. 길이를 검증하는 로직, 2. 숫자로만 이루어져 있는지 판단하는 로직)은 
**비슷한 추상화 수준**을 갖추면 조금 더 읽기 쉬운 코드가 되지 않을까 생각합니다 ..!!  
3. 입력값의 중복을 검증하는 로직을 깜빡하고 놓치신 것 같아요!

```suggestion

const isOnlyNumber = ( array ) => array.some((item) => isNaN(item)); 
const isThreeDigit = (array) => array.length === 3;

const ValidateInput = (arrayInput) => {
    if(!isOnlyNumber){
        printInputTypeError();
    }
    if(!isThreeDigit) {
        printInputLengthError();
    }
    
    /** 입력값 중복 여부 검사 로직 추가하기 */
}
```

Reply:
1. 이름 작명을 신경 쓴다고 했는데 또 놓친 게 있네요ㅠㅠ 되짚어 주신 점 감사드립니다!
2. 말씀해 주시니 눈에 고쳐야 할 점이 눈에 확 띄네요..! 피드백 감사합니다!
3. 코딩을 할 때 중복 입력도 하나의 게임 전략이지 않을까 생각했었는데 지금 다시 생각해 보니 중복 검사를 하는 게 더 좋은 방법 같네요! 피드백 감사합니다!